Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -176,14 +177,24 @@ private void Initialize()
                 Utilities.Editor.InputMappingAxisUtility.CheckUnityInputManagerMappings(Definitions.Devices.ControllerMappingLibrary.UnityInputManagerAxes);
 #endif
 
-                //Enable Input (example initializer)
                 AddManager(typeof(IMixedRealityInputSystem), Activator.CreateInstance(ActiveProfile.InputSystemType) as IMixedRealityInputSystem);
             }
 
+            // If the Speech system has been selected for initialization in the Active speech profile, enable it in the project
+            if (MixedRealityManager.Instance.ActiveProfile.IsSpeechCommandsEnabled)
+            {
+                AddManager(typeof(IMixedRealitySpeechSystem), Activator.CreateInstance(ActiveProfile.SpeechCommandsProfile.SpeechSystemType, "Speech Manager", (uint)10) as IMixedRealitySpeechSystem);
+            }
+
+            // If the Dictation system has been selected for initialization in the Active speech profile, enable it in the project
+            if (MixedRealityManager.Instance.ActiveProfile.IsDictationEnabled)
+            {
+                AddManager(typeof(IMixedRealityDictationSystem), Activator.CreateInstance(ActiveProfile.DictationSystemType, "Dictation Manager", (uint)10) as IMixedRealityDictationSystem);
```

Comment:
Do we need to cast to uint here?

Reply:
Yes, else the activator does not recognise it.  It will assume the value is an Int and not find the constructor.